### PR TITLE
Remove `ManagedClusterConditionAvailable`

### PR DESCRIPTION
**Pull request by dhaiducek**
Setting `ManagedClusterConditionAvailable` in a `PlacementRule` is tumultuous in Policy if managed clusters become unavailable because the Policy will be removed from the managed cluster.

### CODE REVIEW

**Comment by dhaiducek** [comment]
(This PR is large because I used `yq`, so it reformatted files that didn't follow its conventions.)

File: community/CM-Configuration-Management/policy-oauth-ldapsync.yaml [hunk 1]
```diff
@@ -109,7 +109,7 @@ spec:
               objectDefinition:
                 apiVersion: v1
                 data:
-                  ldap-sync.yml: >
+                  ldap-sync.yml: |
```
Comment (mprahl) [inline community/CM-Configuration-Management/policy-oauth-ldapsync.yaml]: I'm wondering why it changed this string. I'd prefer to revert this plus the random new line after `augmentedActiveDirectory`.

Comment (dhaiducek) [reply community/CM-Configuration-Management/policy-oauth-ldapsync.yaml]: That was me. The underlying `go-yaml` parser in `yq` translates multiline strings to show as they would after the multiline marker is applied, so I fiddled with it to get it to show properly, but I forgot to flip it back. I'll revert it.

**Review by mprahl — APPROVED** [review]

**Comment by mprahl** [comment]
Thanks for doing this @dhaiducek!

**Pull request merged**